README.md

```diff
@@ -128,12 +128,7 @@ task = device.run(bell, s3_folder, poll_timeout_seconds=86400)  # 1 day
 print(task.result().measurement_counts)
 ```
 
-Specify which quantum computer hardware to use by changing the value of the `device_arn` to the value for quantum computer to use:
-- **IonQ** "arn:aws:braket:::device/qpu/ionq/ionQdevice"
-- **Rigetti** "arn:aws:braket:::device/qpu/rigetti/Aspen-8"
-- **D-Wave** (See the next section in this document for more information about using D-Wave.)
-    - **D-Wave 2000Q_6** "arn:aws:braket:::device/qpu/d-wave/DW_2000Q_6"
-    - **D-Wave Advantage_system1** "arn:aws:braket:::device/qpu/d-wave/Advantage_system1"
+Specify which quantum hardware device to use by specifying its ARN as the value of the `device_arn` argument. A list of all available quantum devices and their features can be found in the Amazon Braket [developer guide](https://docs.aws.amazon.com/braket/latest/developerguide/braket-devices.html).
```

Comment:
To select a quantum hardware device, specify its ARN as the value of the `device_arn` argument. A list of available quantum devices and their features can be found in the [Amazon Braket Developer Guide](https://docs.aws.amazon.com/braket/latest/developerguide/braket-devices.html).

(Note to writer: The actual title of the document is as shown. Other edits for clarity and grammar.)